### PR TITLE
Use component-specific log4j config files

### PR DESCRIPTION
**Pull request by gracechensd**
**Context:**
Based off of #47 
In many CP images we already have a [log4j property ](https://github.com/confluentinc/kafka-images/blob/703e800d87acaf060b6eadd8c8bcf06470898579/zookeeper/Dockerfile.ubi8#L74)file at `/etc/$COMPONENT/log4j.properties`, which is configurable by customers and documented [here](https://docs.confluent.io/platform/current/installation/docker/operations/logging.html#log4j-log-levels). We want CP images to use said `log4j.properties` files.

**Changes**
Use component-specific log4j config files which are found at `/etc/$COMPONENT/log4j.properties`, or `/etc/cp-base-new-/log4j.properties` by default. `$COMPONENT` is an environment variable which is defined by individual CP repos, e.g. [kafka-mqtt here](https://github.com/confluentinc/kafka-mqtt-images/blob/master/kafka-mqtt/Dockerfile.ubi8#L41)

### CODE REVIEW

**Review by xjin-Confluent — APPROVED** [review]
Thanks for the hard work to put this together.

**Comment by gracechensd** [comment]
Will update [common-docker](https://github.com/confluentinc/common-docker/blob/master/pom.xml#L54)'s versions after merging this and kick off new image builds

File: confluent/docker_utils/cub.py [hunk 1]
```diff
@@ -394,6 +404,7 @@ def ensure_topic(config, file, timeout, create_if_not_exists):
     """
     cmd_template = """
              java {jvm_opts} \
+                 -Dlog4j.configuration=file:/{log4j_config} \
```
Comment (xjin-Confluent) [inline confluent/docker_utils/cub.py:407]: There is an extra "/" after "file:".

File: confluent/docker_utils/cub.py [hunk 2]
```diff
@@ -93,6 +93,12 @@ def __request(host, port, secure, ignore_cert, username='', password='', path=''
     auth = HTTPBasicAuth(username, password) if (username or password) else None
     return requests.get(url, verify = not ignore_cert, auth = auth)
 
+def log4j_config_file():
+    if (os.environ.get("COMPONENT")):
+        return "/etc/" + os.environ.get("COMPONENT") + "/log4j.properties"
```
Comment (xjin-Confluent) [inline confluent/docker_utils/cub.py:98]: Nit: Can we further make sure this file exists?

**Pull request merged**